dask_cudf/io/csv.py

```diff
@@ -51,4 +64,29 @@ def _read_csv(fn, dtypes=None, **kwargs):
         df = dd.core.make_meta(dtypes)
         cdf = cudf.from_pandas(df)
 
-    return cdf
\ No newline at end of file
+    return cdf
+
+
+def read_csv_with_compression(path, **kwargs):
+    """Read CSV with Compression (gzip/zip)
+
+    Parameters
+    ----------
+    path : str
+        path to files (support for glob)
+    """
+    filenames = sorted(glob(str(path)))
+    name = "read-csv-w-compression-" + tokenize(
```

Comment:
I recommend just `read-csv`.  These names are what appears in the diagnostics like progress bars.  It's best to keep them short.

Reply:
```suggestion
    name = "read-csv-" + tokenize(
```